Dockerfile

```diff
@@ -55,6 +54,10 @@ COPY --from=builder /build/unlockerd /
 
 RUN install /tdex /bin
 RUN install /unlockerd /bin
+# Prevents `VOLUME $HOME/.tdex-daemon/` being created as owned by `root`
+RUN useradd -ms /bin/bash user
```

Comment:
shouldn the user be created before the install of binaries?

Reply:
@altafan

Reply:
When the user is created shouldn't be so important. Btw, at the beginnig I was creating and setting the user before installing the binaries and this was causing errors when building the image.